.github/workflows/helm-test.yml

```diff
@@ -0,0 +1,135 @@
+name: Test lighty application
+on:
+  workflow_dispatch:
+    inputs:
+      app-name:
+        description: Name of the application in /lighty-applications
+        default: lighty-rnc-app
+        required: true
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    env:
+      app-aggregator-dir: lighty-applications/${{ github.event.inputs.app-name }}-aggregator
+      SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+      SONAR_ORGANIZATION: ${{ secrets.SONAR_ORGANIZATION }}
+      SONAR_PROJECT_KEY: ${{ secrets.SONAR_PROJECT_KEY}}
+    name: Setup, Install and Test
+    steps:
+      - name: Clone Repository
+        uses: actions/checkout@v2
+      - name: Set up JDK 1.11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
+      - name: Cache Maven packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Maven install + build docker image (skip test)
+        run: mvn install -Pdocker -DskipTests -B -V -Psource-quality
+      - name: Maven test + SonarCloud
+        if: ${{ env.SONAR_TOKEN != 0 }}
+        run: mvn -B verify org.sonarsource.scanner.maven:sonar-maven-plugin:sonar
```

Comment:
`mvn -B verify` should not be needed
run only appropriate mvn phase, install or maybe package should be enough